src/EnergyPlus/HVACSingleDuctInduc.cc

```diff
@@ -1591,6 +1599,24 @@ namespace HVACSingleDuctInduc {
         return YesNo;
     }
 
+    void IndUnitData::ReportIndUnit()
+    {
+        // Purpose: this subroutine for reporting
+
+        // set zone OA volume flow rate
+        this->CalcOutdoorAirVolumeFlowRate();
+    }
+
+    void IndUnitData::CalcOutdoorAirVolumeFlowRate()
+    {
+        // calculates zone outdoor air volume flow rate using the supply air flow rate and OA fraction
+        if (this->AirLoopNum > 0) {
+            this->OutdoorAirFlowRate = (DataLoopNode::Node(this->PriAirInNode).MassFlowRate / DataEnvironment::StdRhoAir) * DataAirLoop::AirLoopFlow(this->AirLoopNum).OAFrac;
+        } else {
+            this->OutdoorAirFlowRate = 0.0;
+        }
+    }
+
```

Comment:
The two new member functions added for FourPipeInduction air terminal unit.

Reply:
Yeah, the repeated nature of these functions is a bit off-putting, but I see that they are not completely identical, as different structures use different names.  In an ideal world, all these structs would share some base class that could handle this common behavior using a base class method.  But that's another day!